kbfsfuse/main.go

```diff
@@ -80,6 +80,13 @@ func start() *libfs.Error {
 	} else {
 		mountDir = flag.Arg(0)
 	}
+	dir, err := os.Stat(mountDir)
+	if err != nil || !dir.IsDir() {
+		err = os.MkdirAll(mountDir, os.ModePerm)
```

Comment:
`os.ModePerm` seems kind of permissive.  Why not `0700`?

Reply:
Shouldn't this be restricted by umask, and additionally by the permissions of the parent directory (which in the case of keybase's XDG_RUNTIME_DIR subdirectory are already 700, on top of 700 for the XDG_RUNTIME_DIR itself).